src/Command/Dispatcher/DispatchLabelsCommand.php

```diff
@@ -54,12 +58,20 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $this->io->title('Dispatch labels for all sonata projects');
 
+        $processor = new Processor();
+        $config = $processor->processConfiguration(new LabelsConfiguration(), [
+            'sonata' => Yaml::parse(file_get_contents(__DIR__.'/../../../config/labels.yaml')),
```

Comment:
```suggestion
            'sonata' => Yaml::parseFile(__DIR__.'/../../../config/labels.yaml'),
```

Reply:
Will fix it 👍

Reply:
https://github.com/sonata-project/dev-kit/pull/1005